src/FakeItEasy/Core/AssemblyExtensions.cs

```diff
@@ -14,16 +14,7 @@ public static bool ReferencesFakeItEasy(this Assembly assembly)
         {
             Guard.AgainstNull(assembly, nameof(assembly));
 
-#if FEATURE_REFLECTION_GETASSEMBLIES
             return assembly.GetReferencedAssemblies().Any(r => r.FullName == TypeCatalogue.FakeItEasyAssembly.FullName);
-#else
-            var fakeItEasyLibraryName = TypeCatalogue.FakeItEasyAssembly.GetName().Name;
-            var context = Microsoft.Extensions.DependencyModel.DependencyContext.Default;
-
-            var runtimeLib = context.RuntimeLibraries
-                .SingleOrDefault(library => string.Equals(library.Name, assembly.Name(), System.StringComparison.Ordinal));
-            return runtimeLib != null && runtimeLib.Dependencies.Any(dependency => string.Equals(dependency.Name, fakeItEasyLibraryName, System.StringComparison.Ordinal));

```

Comment:
> Ooops. Just realized we're talking about removed code, so there's nothing to change

Ah, sorry about that... I'm colorblind, and sometimes I mistake removed code for added code.